pages/assessment.tsx

```diff
@@ -167,6 +165,16 @@ const Assessment: NextPage = () => {
     }
   };
 
+  const findAndSetCurrentPage = (assessments: AssessmentType[]) => {
```

Comment:
Can this be called in `handleSetMaxItems()` as well? 
Replacing line 158-164.
The function would then need to take in `value` which basically is maxItemsPerPage